src/Symfony/Component/Filesystem/FS.php

```diff
@@ -0,0 +1,322 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Filesystem;
+
+use Symfony\Component\Filesystem\Exception\FileNotFoundException;
+use Symfony\Component\Filesystem\Exception\IOException;
+
+/**
+ * Provides basic utility to manipulate the file system.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class FS
```

Comment:
Can't we use a more general classname? This seems so short and weird to have in the codebase.

Reply:
`FileSystem`? 😄 

Jokes aside, it was a suggestion of @nicolas-grekas and I quite like it. While not ideal, it allows to provide a simple implementation without doing to much changes to the `Filesystem` class/API.

An alternative would be to go for functions instead. It would love to do that personally but I'm afraid that solution is not ideal due to PHP not having a proper function autoloading mechanism